raml-validate.js

```diff
@@ -377,17 +392,25 @@ module.exports = function () {
      */
     return function (model) {
       model = model || {}
-
-      // Map all validations to their object and filter for failures.
-      var errors = Object.keys(validations).map(function (param) {
-        var value = model[param]
-        var validation = validations[param]
-
-        // Return the validation result.
-        return validation(value, param, model)
-      }).filter(function (validation) {
-        return !validation.valid
-      })
+      var errors = []
+
+      if (isObjectType || RAMLVersion !== 'RAML10') {
```

Comment:
maybe revert the order of this `if...else` so that it's consistent with lines 377-384. in any case, might be more explicit to do `RAMLVersion === 'RAML08'`...